migrations/20200702154701-bulk_entity_sample_csv_upload.js

```diff
@@ -0,0 +1,68 @@
+
+const fs = require('fs');
+
+const https = require('https');
+const request = require('request');
+
+module.exports = {
+  async up(db) {
+
+    global.migrationMsg = "Upload bulk entity sample csv to cloud"
+   
+    let uploadfileInfo ={ name: "entities.csv", path: process.env.BULK_ENTITIES_SAMPLE_CSV_PATH };
+
+    let uploadFolderPath = "bulkUpload/";
+    
+    let endPoint = "";
+
+    if(process.env.CLOUD_STORAGE == "AWS"){
+      endPoint = "/cloud-services/aws/uploadFile";
+    }else if(process.env.CLOUD_STORAGE == "GC"){
+      endPoint = "/cloud-services/gcp/uploadFile";
+    }else if(process.env.CLOUD_STORAGE == "AZURE"){
+      endPoint = "/cloud-services/azure/uploadFile";
+    }
+
+    let bucketName = process.env.STORAGE_BUCKET;
+
+    let kendraBaseUrl =
+      process.env.KENDRA_SERIVCE_HOST +
+      process.env.KENDRA_SERIVCE_BASE_URL +
+      process.env.URL_PREFIX;
+
+    let response = await apiCall(kendraBaseUrl+endPoint,uploadfileInfo);
```

Comment:
@rakeshSgr - Can we not just use the code in kendra service helper created in this repo to upload, rather than copying the same code in different migration files